### ISSUE TITLE
Subatomic Particle and structure ontology not disjoint

### ISSUE THREAD

**Issue by bitbarbie** [ebi-chebi/ChEBI#3169]
The structural ontology, of which chemical entity (CHEBI:24431) is the root, was supposed to be disjoint from the subatomic particle ontology (root CHEBI:36342). I found a hand full of examples where it is possible to reach both roots via is_a relations (example and full list below). Is there a wrong connection in the graph?

CHEBI:30220
CHEBI:30216
CHEBI:24636
CHEBI:29233
CHEBI:29234

![subatomic_particle_example](https://cloud.githubusercontent.com/assets/3034317/16767386/7b3c559c-483f-11e6-98a2-2bc3d554e33c.png)

**Comment by G-Owen** [comment]
I don't think that we specify the structural and subatomic particle ontologies are disjoint? While it is generally true that  monoatomic monocations such as sodium(1+), copper(1+), etc., are not subatomic particles,  the generalization breaks down with monoatomic hydrogen and its isotopes. Removing an electron from a hydrogen atom, H, gives a proton, H(+) - which is a subatomic particle.
